docs/Playing/how-to-play.rst

```diff
@@ -1,224 +1,366 @@
+takes more moves for your :unit:`Settlers` to reach the planned city center of the city. In Smallpoxing you
+only have to move a few tiles. With Largepoxing you have to move twice the distance and time (in turns).
+
+There is a third stragey. It is not called “Midpoxing” per se, but the strategy is similar to a middle ground
+between the Smallpoxing and Largepoxing strategies. Here is an example of a Midpoxing strategy:
+
+.. _Midpoxing:
+.. figure:: /_static/images/how-to-play/midpoxing.png
+    :scale: 75%
+    :align: center
+    :alt: Midpoxing
+    :figclass: align-center
+
+    Midpoxing
+
+
+Recall that there are a number of factors to consider when it comes to city planning and placement:
+
+#. Distance from the Capital.
+#. Available Resources on tiles around the potential city centers.
+#. Locations of freshwater from rivers and lakes.
+#. No wasted tiles. Force overlap of any kind so that every tile is available.
+
+Taking Care of the Cities
+-------------------------
+
+Every city has a group of citizens. The number of citizens depends on the city's population. When you click on
+a city, you can see how the land around the city is being managed. You can assign the city's citizens to
+manage the land, or they can be specialists that contribute to your civilization in other ways. Especially at
+the start of the game, care should be taken to ensure that the citizens are employed so that they maximize
+food, trade, and then production.
+
+If too much food is being generated, a citizen can be taken off the land by clicking on the occupied land
+tile. This citizen can then be transferred to a specialist. There are three specialists: Taxmen (collect
+gold), Entertainers (create happiness), or Scientist (create bulbs). By default, Freeciv21 will add an
+Entertainer when you take a citizen off the land.
+
+If you click on the central tile of the land (the "City Center"), the citizens will be rearranged to maximize
+food production. You will also want to look at the level of food in the Granary and the amount of surplus food
+the city is producing each turn. The city will lose excess food at turn change. For example: The Granary in
+the city has 18/20 food (needs two food to grow to next city size) and the city is currently producing +4 food
+surplus each turn. This means that at turn change you will lose 2 food as waste at turn change. This is a good
+opportunity to move the citizens around to get food to +2 surplus. This could be accomplished by taking a
+citizen off a tile producing 2 food and turn it into a Taxman for a turn to get gold. At turn change, open the
+city and restore the citizen to farming. This kind of individual city management style is called
+"micro-management" and is a very powerful mechanism of game play.
+
+The golden rule of taking care of a city is that there should be at least as many happy citizens as unhappy
+citizens. A city where this is not so falls into disorder. Such cities are labeled with a raised fist or a
```

Comment:
```suggestion
citizens. A city where this is not the case falls into disorder. Such cities are labeled with a raised fist or a
```

Same meaning, simpler wording (I believe). Just an optional suggestion, can be ignored

Reply:
i agree, it is simpler. good catch